### ISSUE TITLE
OkCoin ETC/USD - Unable to read wallet data positions

### ISSUE THREAD

**Issue by Jorman** [ctubio/Krypto-trading-bot#124]
Hi, on OkCoin I've 0.189 etc so about 3$, a slippage from a btc change but ...

I start the bot but I've this error, is my fault or something is happen?
```(noaccess)
K.linux.57
- pair: etc_usd
15:0:46.944464 GW OkCoin Unable to read wallet data positions.
15:0:50.66441 GW OkCoin WS Error, reconnecting in 5s..
15:00:54.459 stdev Unable to compute value
terminate called after throwing an instance of 'std::out_of_range'
  what():  unresolved reference token '4'
==23959==
==23959==    by 0x9D61D50: nlohmann::basic_json<std::map, std::vector, std::string, bool, long, unsigned long, double, std::allocator, nlohmann::adl_serializer>::json_pointer::get_unchecked(nlohmann::basic_json<std::map, std::vector, std::string, bool, long, unsigned long, double, std::allocator, nlohmann::adl_serializer>*) const (in /home/lello/K/app/server/lib/K.linux.57.node)
==23959==    by 0x9D6B1EB: K::GwOkCoin::book()::{lambda(uWS::WebSocket<false>*, char*, unsigned long, uWS::OpCode)#4}::operator()(uWS::WebSocket<false>*, char*, unsigned long, uWS::OpCode) const (in /home/lello/K/app/server/lib/K.linux.57.node)
Killed
```
Jo

**Comment by ctubio** [comment]
seems an error at K::GwOkCoin::book(), but im not able to reproduce it; for you only happens at OkCoin ETC/USD or also at BTC market?

**Comment by ctubio** [comment]
seems like in OkCoin the market trades doesnt have `side` in ETC/USD, but BTC market have trades with `side` in OkCoin...

im not using ETC/USD, so this may be fixed by me for free only after i had finnished all my other migration tasks

**Comment by Jorman** [comment]
Tnx for your time. Like I said I usually don't use etc but was from a slippage order so I tried to use the small amount to another pair

**Comment by ctubio** [comment]
i belive this was already fixed in latest commits; feel free to open a new issue if you still have issues on Okcoin
